pkg/agent/flowexporter/connections/connections.go

```diff
@@ -122,13 +120,21 @@ func (cs *connectionStore) addOrUpdateConn(conn *flowexporter.Connection) {
 	cs.mutex.Lock()
 	defer cs.mutex.Unlock()
 	if exists {
```

Comment:
We don't have special handling for that case because we need to store the connection and export related flow records. The connection will be deleted when it no longer exists in conntrack.